base_layer/core/src/transactions/key_manager/inner.rs

```diff
@@ -241,7 +241,7 @@ where TBackend: KeyManagerBackend<PublicKey> + 'static
     }
 
     /// Search the specified branch key manager key chain to find the index of the specified private key.
-    async fn find_private_key_index(&self, branch: &str, key: &PrivateKey) -> Result<u64, KeyManagerServiceError> {
+    pub async fn find_private_key_index(&self, branch: &str, key: &PrivateKey) -> Result<u64, KeyManagerServiceError> {
```

Comment:
I don't remember why this change was made. See about reverting it if it's not necessary.